Extension/src/LanguageServer/settings.ts

```diff
@@ -73,6 +73,29 @@ class Settings {
         }
         return undefined;
     }
+
+    public getWorkspaceFolderValue<T>(section: string): T | undefined {
```

Comment:
After our iterations, I think this function has now become obsolete.  I believe it will behave the same as the official `get<T>` API now.

Reply:
Not relevant due to scope change, will remove these